2_Operations/work-process.md

```diff
@@ -4,6 +4,7 @@
 ## Schedule
 
  * Every weekday morning we hold a daily standup meeting.
+ * Every weekday each Nitean needs to check GitHub notifications.
```

Comment:
Every weekday Niteans should check their [GitHub notifications](https://github.com/notifications) so that they don't miss things that involve them.